alacritty/src/display/window.rs

```diff
@@ -345,6 +349,13 @@ impl Window {
 
     pub fn set_transparent(&self, transparent: bool) {
         self.window.set_transparent(transparent);
+        #[cfg(target_os = "macos")]
+        set_titlebar_transparent(&self.window, transparent);
+    }
+
+    pub fn set_background_opacity(&self, _opacity: f32) {
+        #[cfg(target_os = "macos")]
+        set_background_opacity(&self.window, _opacity as _)
     }
```

Comment:
Is there ever a scenario where you'd want to call `set_background_opacity` without setting transparency? Seems like the two functions can just be combined.

Reply:
They probably could be merged, but I think it makes sense to keep them as separate function to self-document specifically what it is that they do.